### PR TITLE
MTP-1666 mtp 1657 svelte part for a vise liste av regelverksprosesser pa en temaside

### PR DESCRIPTION
**Pull request by petterkoding**
use with https://github.com/Mattilsynet/mattilsynet-web/pull/550

### CODE REVIEW

File: mt-kit/core/svelte/src/lib/svelte/components/Published.svelte [hunk 1]
```diff
@@ -6,6 +6,7 @@
   export let lang = 'NO-nb'
   export let publishedText = 'Publisert'
   export let lastPublishedText = 'Faglig oppdatert'
+  export let dtClass
```
Comment (lha047) [inline mt-kit/core/svelte/src/lib/svelte/components/Published.svelte:9]: Add default empty string so it doesn't put a undefined class in the string

Comment (petterkoding) [reply mt-kit/core/svelte/src/lib/svelte/components/Published.svelte]: ok

File: mt-kit/core/svelte/src/lib/svelte/components/Published.svelte [hunk 2]
```diff
@@ -6,6 +6,9 @@
   export let lang = 'NO-nb'
   export let publishedText = 'Publisert'
   export let lastPublishedText = 'Faglig oppdatert'
+  export let dtIcon = ''
```
Comment (lha047) [inline mt-kit/core/svelte/src/lib/svelte/components/Published.svelte:9]: On components i would only use class for the root component.
To target other elements use eg. dtClass. Dont think you need two variables for putting different classes. Would only use that when there is theming sizing etc. and that is a part of the class name. I would not use the same name incase someone later dont want the same styling

**Review by lha047 — CHANGES_REQUESTED** [review]
See comment

**Review by lha047 — APPROVED** [review]
Nice :D

**Pull request merged**